templates/order.php

```diff
@@ -80,6 +80,9 @@
                 <span class="ng-cloak bnomics-time-left" ng-hide="order.status != -1">{{clock*1000 | date:'mm:ss' : 'UTC'}} min left to pay your order</span>
               </div>
         <!-- Blockonomics Credit -->
+			<div class="bnomics-how-to-pay">
```

Comment:
Let's remove the changes from the how-to-pay branch.

Reply:
done.